src/transformers/models/wav2vec2/configuration_wav2vec2.py

```diff
@@ -163,6 +164,16 @@ def __init__(
         mask_time_length=10,
         mask_feature_prob=0.0,
         mask_feature_length=10,
+        quantize_targets=False,
+        num_latent_vars=320,
+        num_latent_groups=2,
+        latent_temp=(2.0, 0.5, 0.999995),
+        logit_temp=0.1,
```

Comment:
what does `temp` stand for => temporal? what exactly is `logit_temp`?

Reply:
The `temp` stands for `temperature`, I've updated the names to be more descriptive now, and added docstrings for them
The values are `(max temp, min temp, annealing factor)` 
The temperature starts at the max value (2.0) and can be annealed at every batch update by a given factor
using a custom callback:
```python
class GumbelTempCallback(TrainerCallback):
    def on_step_begin(self, args, state, control, model: Wav2Vec2ForPreTraining, **kwargs):
        model.update_gumbel_temperature(state.global_step)

    trainer = Trainer(...)
    trainer.add_callback(GumbelTempCallback)
```

Reply:
This should probably go into the model docs